corehq/warehouse/transforms/sql/location_dim.sql

```diff
@@ -0,0 +1,97 @@
+-- First insert all records into location dim from staging tables without foreign keys
+INSERT INTO {{ location_dim }} (
```

Comment:
we'll need to handle updates here as well eventually, or is that coming for all dims in a later pr